src/panels/lovelace/hui-root.ts

```diff
@@ -0,0 +1,583 @@
+                    .hass="${this.hass}"
+                    .notificationsOpen="{{notificationsOpen}}"
+                    .notifications="${this._notifications}"
+                  ></hui-notifications-button>
+                  <ha-start-voice-button
+                    .hass="${this.hass}"
+                  ></ha-start-voice-button>
+                  <paper-menu-button
+                    no-animations
+                    horizontal-align="right"
+                    horizontal-offset="-5"
+                  >
+                    <paper-icon-button
+                      icon="hass:dots-vertical"
+                      slot="dropdown-trigger"
+                    ></paper-icon-button>
+                    <paper-listbox
+                      @iron-select="${this._deselect}"
+                      slot="dropdown-content"
+                    >
+                      ${
+                        this._yamlMode
+                          ? html`
+                              <paper-item @click="${this._handleRefresh}"
+                                >Refresh</paper-item
+                              >
+                            `
+                          : ""
+                      }
+                      <paper-item @click="${this._handleUnusedEntities}"
+                        >Unused entities</paper-item
+                      >
+                      <paper-item @click="${this._editModeEnable}"
+                        >${
+                          this.localize("ui.panel.lovelace.editor.configure_ui")
+                        }</paper-item
+                      >
+                      ${
+                        this._storageMode
+                          ? html`
+                              <paper-item
+                                @click="${this.lovelace!.enableFullEditMode}"
+                                >Raw config editor</paper-item
+                              >
+                            `
+                          : ""
+                      }
+                      <paper-item @click="${this._handleHelp}">Help</paper-item>
+                    </paper-listbox>
+                  </paper-menu-button>
+                </app-toolbar>
+              `
+        }
+
+        ${
+          this.lovelace!.config.views.length > 1 || this._editMode
+            ? html`
+                <div sticky>
+                  <paper-tabs
+                    scrollable
+                    .selected="${this._curView}"
+                    @iron-activate="${this._handleViewSelected}"
+                  >
+                    ${
+                      this.lovelace!.config.views.map(
+                        (view) => html`
+                          <paper-tab>
+                            ${
+                              view.icon
+                                ? html`
+                                    <ha-icon
+                                      title="${view.title}"
```

Comment:
`.title`

Reply:
It was set in Polymer like title$ which meant set it as attribute. That makes sense so the browser will show the tooltip